inexor/ui/CefSubsystem.cpp

```diff
@@ -42,7 +41,13 @@ void CefSubsystem::tick() {
 
 void CefSubsystem::initialize(int argc, char **argv)
 {
-    spdlog::get("global")->info("CefSubsystem::initialize() --> CefInitialize()");
+	spdlog::get("global")->info("CefSubsystem::initialize() --> CefInitialize()");
+
+    if(argc < 2) return;
+    std::string port = argv[1];
+
+    cef_app = new InexorCefApp(port, 1920, 1080);
```

Comment:
screen_manager.scr_w and screen_manager.scr_h could be used here instead of hardcoding it, couldnt it?
(Since we have that stuff from RPC/flex before we initialize cef)